tests/test_flake8_trio.py

```diff
@@ -0,0 +1,66 @@
+import ast
+import os
+import site
+import sys
+import unittest
+from pathlib import Path
+from typing import Any, List, Tuple, Type
+
+from hypothesis import HealthCheck, given, settings
+from hypothesmith import from_grammar
+
+from flake8_trio import TRIO100, Error, Plugin, Visitor
+
+
+class Flake8TrioTestCase(unittest.TestCase):
+    def errors(self, *errors: Error) -> List[Tuple[int, int, str, Type[Any]]]:
+        return [e.values() for e in errors]
+
+    def test_tree(self):
+        plugin = Plugin(tree=ast.parse(""))
+        errors = list(plugin.run())
+        self.assertEqual(errors, [])
+
+    def test_trio100(self):
+        filename = Path(__file__).absolute().parent / "trio100.py"
+        plugin = Plugin(filename=str(filename))
+        errors = list(plugin.run())
+        expected = self.errors(
+            TRIO100(3, 5, "trio.move_on_after"),
+            TRIO100(23, 15, "trio.fail_after"),
+        )
+        self.assertEqual(errors, expected)
+
+    @unittest.skipIf(sys.version_info < (3, 9), "requires 3.9+")
+    def test_trio100_py39(self):
+        filename = Path(__file__).absolute().parent / "trio100_py39.py"
+        plugin = Plugin(filename=str(filename))
+        errors = list(plugin.run())
+        expected = self.errors(
+            TRIO100(7, 8, "trio.fail_after"),
+            TRIO100(12, 8, "trio.fail_after"),
+            TRIO100(14, 8, "trio.move_on_after"),
+        )
+        self.assertEqual(errors, expected)
+
+
+class TestFuzz(unittest.TestCase):
```

Comment:
I'd skip this by default - as you note it's pretty slow, and I think running it by hand occasionally is plenty.

Reply:
Adding a `pytest.mark.fuzz` to them, so could maybe add `--runfuzz` to the release test (which I think only you can add?).